catalog/build_ci/B004.md

```diff
@@ -1,8 +1,8 @@
 ### B004
 
 |**Code**           | **B004** |
-|**Title**          | Prevent Compiler Warning Automatically. |
-|**Description**    | |
+|**Title**          | Prevent Compiler Warning. |
+|**Description**    | Some procedures prevent merging codes with compiler warning.|
```

Comment:
As an example, a project may have compiler warnings on main branch, but prevent new changes that have such warnings; does the project meet this item in this case? If so, I recommend clarifying it in the description. like:
```suggestion
|**Description**    | No compiler warnings exist on main branch and some procedures prevent merging code with such warnings. |
```

Reply:
I replaced your suggestion.